fab/fabfile.py

```diff
@@ -611,7 +650,7 @@ def preindex_complete():
                 _execute_with_timing(stop_celery_tasks)
             _execute_with_timing(_migrate)
         else:
-            print(blue("No migration required, skipping."))
+            print blue("No migration required, skipping.")
```

Comment:
most of the file uses print statments and there's no `from __future__ import print_function` at the top.  I just switched this for consistency, because what was happening was "print the following single-element tuple, but without a space, so it looks like a function call".